examples/sample-pub/publisher.js

```diff
@@ -144,6 +144,43 @@ function startFlowAuto() {
     return;
   }
 
+  if (flow == 'metering') {
+    /* eslint-disable */
+    const timestamp = 1597686771;
+
+    whenReady((subscriptions) => {
+      subscriptions
+        .getEntitlements({
+          metering: {
+            state: {
+              id:
+                'user5901e3f7a7fc5767b6acbbbaa927d36f5901e3f7a7fc5767b6acbbbaa927',
+              standardAttributes: {
+                registered_user: {
+                  timestamp,
+                },
+              },
+              customAttributes: {
+                newsletter_subscriber: {
+                  timestamp,
+                },
+              },
+            },
+          },
+        })
+        .then((entitlements) => {
+          if (entitlements.enablesThisWithGoogleMetering()) {
+            console.log(
+              'Consuming Google metering entitlement from Publisher JS'
```

Comment:
Can you please add more detail / comments / links to docs / etc? If I'm a developer on the publisher's side I won't understand what this does and why I want to put it in my code.

Reply:
For sure, I added comments in `publisher.js` and `entitlements-flow.md`